scipy/stats/__init__.py

```diff
@@ -161,7 +161,7 @@
    multivariate_normal    -- Multivariate normal distribution
    matrix_normal          -- Matrix normal distribution
    multivariate_beta      -- Multivariate beta distribution (Dirichlet)
-   dirichlet              -- Dirichlet (deprecated; see `multivariate_beta`)
+   dirichlet              -- Dirichlet (for new code, use `multivariate_beta`)
```

Comment:
I had to remove this from the guide to pass the ref guide CI.

Reply:
But that is a problem with the refguide check, not the documentation. Shouldn't the check be fixed?

Reply:
I suppose we can argue with both. If we deprecate something, we don't want to show old things to people as they usually do not care about warnings until they break. Making things less discoverable can help further.

Reply:
With this PR, the `dirichlet` _distribution_ would not be deprecated; only the `pdf` and `logpdf` methods.